src/About/About.js

```diff
@@ -4,71 +4,63 @@ import Box from '@material-ui/core/Box';
 import Container from '@material-ui/core/Container';
 import GridList from '@material-ui/core/GridList';
 import GridListTile from '@material-ui/core/GridListTile';
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom';
+import { tileData } from '../assets/tileData.js';
 
 const useStyles = makeStyles(theme => ({
   root: {
-      display: 'flex',
-      flexWrap: 'wrap',
-      justifyContent: 'space-around',
-      overflow: 'hidden',
-      backgroundColor: theme.palette.background.paper
+    display: 'flex',
+    // flexWrap: 'wrap',
```

Comment:
A quick note to clean up code before submitting the finalized project.

Reply:
🧹🧼🧽 Let's do it